_CodingChallenges/003-snake-game-p5.md

```diff
@@ -42,6 +42,14 @@ contributions:
       name: "Makarand Subhash Lahane"
       url: "https://mklahane.github.io/Game-Development/"
     url: "https://mklahane.github.io/Game-Development/NewSnakeGame"
+    
```

Comment:
There seems to be whitespace here which means the tests are failing and we can't merge it. As soon as it's removed, it should be good to go! 👍